sdk/storage/storage-blob/review/storage-blob.api.md

```diff
@@ -862,8 +861,10 @@ export class BrowserPolicyFactory implements RequestPolicyFactory {
 
 // @public
 export interface CommonOptions {
+    // Warning: (ae-forgotten-export) The symbol "OperationTracingOptions" needs to be exported by the entry point index.d.ts
```

Comment:
This is a deliberately forgotten export, for the record. We will update this to import the core-http type post-GA.